setup.py

```diff
@@ -53,9 +53,7 @@ def get_version(rel_path):
             "moviepy",
             "pygame",
             "opencv-python",
-            "pykinect_azure",
             "matplotlib",
-            "mediapy",
         ],
     },
 )
```

Comment:
I'd suggest add `dearpygui` to default requirements, instead of putting it into `dev`, as this is strictly required by the `demo_player` and people will use it for visualizations.